src/NUnitEngine/nunit.engine/Services/ProjectLoaders/NUnitProject.cs

```diff
@@ -259,6 +254,20 @@ private string GetBasePathForConfig(XmlNode configNode)
             return settings;
         }
 
+        static readonly char[] PATH_SEPARATORS = new char[] { '/', '\\' };
+
+        private string NormalizePath(string path)
+        {
+            char sep = Path.DirectorySeparatorChar;
+
+            if (path != null)
+                foreach (char alt in PATH_SEPARATORS)
```

Comment:
I am going to merge this, but I think that our coding standards should use brackets for all except the simplest one line control statements. This one is four one liners deep which makes it harder to read and easier to mess up if you need to change the code.